v2/tools/generator/internal/codegen/storage/testdata/Test_StorageTypeFactory_GeneratesExpectedResults-v20200101.golden

```diff
@@ -1,170 +0,0 @@
-// Code generated by azure-service-operator-codegen. DO NOT EDIT.
-// Copyright (c) Microsoft Corporation.
```

Comment:
Were these just "dead" golden files that have no test associated with them?

Reply:
Yes, that's right. We must have renamed the test without doing cleanup.